qpc/utils.py

```diff
@@ -218,11 +218,18 @@ def read_server_config():
                       QPC_SERVER_CONFIG, ssl_verify)
             return None
 
-        if (ssl_verify is not None and
-                isinstance(ssl_verify, str) and
-                not os.path.exists(ssl_verify)):
-            log.error('Server config %s has invalid path for ssl_verify %s',
-                      QPC_SERVER_CONFIG, ssl_verify)
+        if (
+            ssl_verify is not None
+            and isinstance(ssl_verify, str)
+            and not os.path.exists(ssl_verify)
+        ):
+            log.error(
+                t(
```

Comment:
@bruno-fs I could not get rid of the t otherwise I would have to refactor the whole document.